src/client/fontengine.cpp

```diff
@@ -219,8 +224,16 @@ gui::IGUIFont *FontEngine::initFont(const FontSpec &spec)
 	assert(spec.mode != FM_Unspecified);
 	assert(spec.size != FONT_SIZE_UNSPECIFIED);
 
+	FontMode mode = spec.mode; // this takes priority over `spec.mode` !
```

Comment:
I think it'd be nicer to modify `spec` to avoid confusion

Reply:
I also found that the URL dialog changed fonts when hovering over buttons. This became too messy, thus moved it to a separate boolean in `FontSpec`.